src/main/charts/confluence/values.yaml

```diff
@@ -736,6 +736,58 @@ confluence:
   #
   additionalJvmArgs: []
 
+  # -- By default Tomcat's server.xml is generated in the container entrypoint from a template
+  # shipped with an official Confluence image. However, server.xml generation may fail if container
+  # is not run as root, which is a common case if Confluence is deployed to OpenShift.
+  #
+  tomcatConfig:
+
+    # -- Mount server.xml as a ConfigMap. Uncomment configuration elements if necessary to
```

Comment:
> Uncomment configuration elements if necessary 

Should the below elements be commented out then?

Reply:
@yzha645 they were, but then I decided to keep them uncommented. I'll change the comment to Override configuration elements if necessary.